src/Plot.tsx

```diff
@@ -38,8 +38,7 @@ function reducer(state: State, action: ReducerActions) {
     }
     case 'removeAxis': {
       const { id } = action.value;
-      const seriesFiltered = state.series.filter((series) => series.id !== id);
-      state.series = seriesFiltered;
+      state.axis[id] = undefined;
```

Comment:
```suggestion
      delete state.axis[id];
```

Reply:
wasn't it more performant the undefined assignation?

Reply:
Performance doesn't matter much here, as we are not really modifying an object but actually calling `immer`'s Proxy (in both cases). I find delete more explicit.

Reply:
that will [break a rule](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-dynamic-delete.md), what do you think of using a Map instead?

Reply:
https://www.youtube.com/watch?v=Xi5rxgDA0Pg
We can disable this rule in the project. Using a Map sounds good too. Just check how to enable it (not globally) in immer as I think it's not enabled by default.

Reply:
Somehow my answer was not sent yesterday, sorry.
I'm OK with using a Map, but be careful because immer doesn't support it by default, you need to enable it: https://immerjs.github.io/immer/docs/installation#pick-your-immer-version
I'm also OK with disabling that ESLint rule in the project: https://www.youtube.com/watch?v=Xi5rxgDA0Pg

Reply:
I'll go for disabling the rule in this case https://youtu.be/L397TWLwrUU